source/vscode/src/webview/webview.tsx

```diff
@@ -214,23 +217,30 @@ function App({ state }: { state: State }) {
           runNames={[]}
         />
       );
+      break;
     case "circuit":
-      return <CircuitPanel {...state.props}></CircuitPanel>;
+      content = <CircuitPanel {...state.props}></CircuitPanel>;
+      break;
     case "help":
-      return <HelpPage />;
+      content = <HelpPage />;
+      break;
     case "documentation":
       // Ideally we'd have this on all web views, but it makes the font a little
       // too large in the others right now. Something to unify later.
       document.body.classList.add("markdown-body");
       document.body.style.fontSize = "0.8em";
-      return (
+      content = (
         <DocumentationView
           fragmentsToRender={state.fragmentsToRender}
           projectName={state.projectName}
         />
       );
+      break;
     default:
       console.error("Unknown view type in state", state);
-      return <div>Loading error</div>;
+      content = <div>Loading error</div>;
+      break;
   }
+
+  return <div style={{ paddingLeft: "8px" }}>{content}</div>;
```

Comment:
We already have a CSS file specifically for the webview body. Why not just add the padding in the CSS there for the entire view, rather than in code on the specific component?

https://github.com/microsoft/qdk/blob/main/source/vscode/src/webview/webview.css